src/dali/plugin/country/jp.py

```diff
@@ -21,3 +21,8 @@
 # JP-specific entry point
 def dali_entry() -> None:
     dali_main(JP())
+
```

Comment:
IDE integration shouldn't require modifying the source code. I usually debug with print, but I did use the VSCode debugger occasionally in the past (not sure how similar it is to Visual Studio). VSCode can be configured to debug DaLI and RP2 without changing the source: I used .venv/bin/dali_us as the entry point.

Reply:
I can create IDE specific dali_us/dali_jp.py files for debugging, but I don't think that will be as simple as just adding this entry point.  I couldn't figure out any way to get visual studio to start using the exe's generated by the "installer".

Reply:
Have you tried using .venv/bin/dali_us and .venv/bin/dali_jp as entry points? They should not require modification.